### PR TITLE
[4.x] JWT Auth configuration properties updated

### LINKED ISSUE #6469: DOCS: 4.x JWT
## Problem Description

We need to update docs for 4.x

- [x] update jwt.adoc for MP

## Guidelines

- [x] Make sure any fixes from SE 3.x are forward ported to 4.x #7515 
- [x] Make sure any fixes from MP 3.x are forward ported to 4.x
- [x] Initial focus is on correctness
- [x] Record migration impact in this issue

## Migration Impact

Summarize incompatibilities between Helidon 3 and Helidon 4.
Summarize actions customers must take to migrate code
OK to reference external specs or documents (for example MicroProfile specs)

### PR DESCRIPTION
**Pull request by Verdent**
### Description

Added missing MP JWT properties.

Fixes #6469

### Documentation

Added missing MP JWT properties.

### CODE REVIEW

File: docs/config/io_helidon_microprofile_jwt.adoc [hunk 1]
```diff
@@ -55,13 +65,13 @@ an important distinction when more than one provider is used
...
 |`atn-token.handler`|string |`Authorization` header with `bearer ` prefix |A handler configuration for inbound token - e.g. how to extract it
 |`atn-token.handler.header`|string |{nbsp} |Name of a header the token is expected in
 |`atn-token.handler.prefix`|string |{nbsp}  |Prefix before the token value (optional)
 |`atn-token.handler.regexp`|string |{nbsp} |Regular expression to obtain the token, first matching group is used (optional)
 |`sign-token`|string |{nbsp} |A group for configuring outbound security
-|`sign-token.jwk.resource.*` |{nbsp} |Configuration of the JWK to use when generating tokens (follows same rules as atn-token.jwk above), this JWK must contain private keys when using asymmetric ciphers
+|`sign-token.jwk.resource`|xref:{rootdir}/config/io_helidon_common_configurable_Resource.adoc[Resource] |{nbsp} |Configuration of the JWK to use when generating tokens (follows same rules as atn-token.jwk above), this JWK must contain private keys when using asymmetric ciphers
```
Comment (ljamen) [inline docs/config/io_helidon_microprofile_jwt.adoc:74]: Just to be consistent with line above:

Configuration of the JWK to use when generating tokens (follows the same rules as atn-token.jwk above). The JWK must contain private keys when using asymmetric ciphers.

**Review by ljamen — COMMENTED** [review]
very small change - otherwise approved.

File: docs/config/io_helidon_microprofile_jwt.adoc (hunk 1 above)
Comment (Verdent) [reply docs/config/io_helidon_microprofile_jwt.adoc:74]: Changed.

**Review by ljamen — APPROVED** [review]
Changes addressed, LGTM.

**Pull request merged**